src/etl/closure_etl.py

```diff
@@ -4,29 +4,26 @@
 import multiprocessing
 
 from etl import ETL
-from transactors import CSVTransactor, Neo4jTransactor
+from transactors import CSVTransactor
+from transactors import Neo4jTransactor
 from .helpers import Neo4jHelper
 
 
 class ClosureETL(ETL):
-    '''Closure ETL'''
-
+    '''Clojure ETL'''
 
     logger = logging.getLogger(__name__)
 
-    # Query templates which take params and will be processed later
-
-    insert_is_a_part_of_closure_query_template = """
+    insert_isa_partof_closure = """
         USING PERIODIC COMMIT %s
         LOAD CSV WITH HEADERS FROM \'file:///%s\' AS row
-
             MATCH (termChild:%sTerm {primaryKey:row.child_id})
             MATCH (termParent:%sTerm {primaryKey:row.parent_id})
             CREATE (termChild)-[closure:IS_A_PART_OF_CLOSURE]->(termParent) """
 
-    retrieve_is_a_part_of_closure_query_template = """
-        MATCH (childTerm:%sTerm:Ontology)-[:PART_OF|IS_A*]->(parentTerm:%sTerm:Ontology) 
-        RETURN childTerm.primaryKey, parentTerm.primaryKey """
+    retrieve_isa_partof_closure = """
+        MATCH (childTerm:%sTerm)-[:PART_OF|IS_A*]->(parentTerm:%sTerm) 
+            RETURN DISTINCT childTerm.primaryKey, parentTerm.primaryKey """
```

Comment:
Was Adding DISTINCT here intended? As that typically kills query performance.

Reply:
this is a change I made in a previous PR - it does typically hurt performance yep, but in this case that query was returning significant levels of duplication and the distinct actually helped reduce overall in the loader (with other refactoring changes as well).